client/src/app/main/dashboard/dashboard-create-ticket/dashboard-create-ticket-page.js

```diff
@@ -33,10 +33,8 @@ class DashboardCreateTicketPage extends React.Component {
     }
 
     onCreateTicketSuccess(result, email, message) {
-        if((this.props.location.pathname !== '/create-ticket')) {
+        if(this.props.location.pathname !== '/create-ticket') {
             history.push(`/dashboard?message=${message}`);
-        } else {
-            setTimeout(() => {history.push('/check-ticket/' + result.data.ticketNumber + '/' + email)}, 1000);
         }
```

Comment:
does the message shows? Can you add a screenshot of how it looks?

Reply:
![Screenshot from 2020-05-06 14-14-13](https://user-images.githubusercontent.com/59095036/81207526-0fc1f780-8fa4-11ea-8f75-937a33a1572d.png)